modules/api/src/main/java/org/apache/ignite/lang/ErrorGroup.java

```diff
@@ -125,62 +129,14 @@ public static String errorMessage(UUID traceId, int code, String message) {
      * @param message Original message.
      * @return New error message with predefined prefix.
      */
-    public static String errorMessage(UUID traceId, String groupName, int code, String message) {
-        return ERR_PREFIX + groupName + '-' + Short.toUnsignedInt(extractErrorCode(code)) + " TraceId:" + traceId
+    public static String errorMessage(String errorPrefix, UUID traceId, String groupName, int code, String message) {
```

Comment:
first, we need to update javadoc. Second, I would propose to do this method package private or even private. The reason is its contract: it accepts `errorPrefix`, `groupName`, and full error `code`, which must correspond to each other

Reply:
fixed